plugins/food_bot_plugin.py

```diff
@@ -74,6 +75,15 @@ def get_week_number():
         return week
 
     @staticmethod
+    def get_meal_time(meal):
+        config = Config()
+
+        if meal == 'breakfast':
+            return config['BREAKFAST_TIME']
+        elif meal == 'lunch':
+            return config['LUNCHTIME']
+
```

Comment:
@andela-ydaniju This looks great. Please add an error message if the meal is not breakfast or lunch. Just in case in the future there is a typo in breakfast or lunch in the db and whoever is debugging starts wondering where the error is coming from. You could log something using logger and also return a message to show that the meal is not breakfast or lunch.